src/System.Private.ServiceModel/src/System/ServiceModel/Security/TrustDriver.cs

```diff
@@ -140,8 +140,5 @@ public virtual string RequestTypeRenew
 
         // RSTR Collection method
         public abstract void WriteRequestSecurityTokenResponseCollection(RequestSecurityTokenResponseCollection rstrCollection, XmlWriter writer);
-
-        // Federation proxy creation
-        public abstract IChannelFactory<IRequestChannel> CreateFederationProxy(EndpointAddress address, Binding binding, KeyedByTypeCollection<IEndpointBehavior> channelBehaviors);
```

Comment:
>public abstract IChannelFactory [](start = 8, length = 31)

Why is this interface removed?  Is there any risk associated with this change?

Reply:
This was the stub for the previous implementation of the WSTrust piece. This was the method which returned a client which could be used to talk to the STS (e.g. an ADFS server). This is now going to be done by the new package that's coming which uses the work that Brent is doing. It was incorrectly pulled over previously and as I was adding another implementation of TrustDriver, I did some cleanup to avoid bringing over another non-used stub.